src/urls/util.ts

```diff
@@ -15,24 +15,12 @@
 import {posix as path} from 'path';
 import {Document} from 'polymer-analyzer';
 
-import {lookupDependencyMapping} from '../manifest-converter';
-
-
 import {ConvertedDocumentFilePath, ConvertedDocumentUrl, OriginalDocumentUrl} from './types';
 
 /**
- * Given an HTML url relative to the project root, return true if that url
- * points to a bower dependency file.
- */
-function isBowerDependencyUrl(htmlUrl: OriginalDocumentUrl): boolean {
-  return htmlUrl.startsWith('bower_components/') ||
-      htmlUrl.startsWith('./bower_components/');
-}
-
-/**
  * Rewrite a url to replace a `.html` file extension with `.js`, if found.
  */
-function replaceHtmlExtensionIfFound(url: string): string {
+export function replaceHtmlExtensionIfFound(url: string): string {
   if (url.endsWith('.html')) {
```

Comment:
this whole bit feels like it should have been written as
```js
return url.replace(/\.html$/, '.js');
```

Do we not favor regex for some performance reason?

Reply:
This went through a few iterations (see some past PRs) but you're right this is much cleaner. 💯